packages/office-ui-fabric-react/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -61,22 +62,24 @@ export class SelectedPeopleList extends BasePeopleSelectedItemsList {
   protected renderItems = (): JSX.Element[] => {
     const { items } = this.state;
     // tslint:disable-next-line:no-any
-    return items.map((item: any, index: number) => this._renderItem(item, index));
+    return items.map((item: IExtendedPersonaProps, index: number) => this._renderItem(item, index));
   };
 
   // tslint:disable-next-line:no-any
-  private _renderItem(item: any, index: number): JSX.Element {
+  private _renderItem(item: IExtendedPersonaProps, index: number): JSX.Element {
     const { removeButtonAriaLabel } = this.props;
+    const expandGroup = this.props.onExpandGroup;
     const props = {
       item,
       index,
       key: item.key ? item.key : index,
       selected: this.selection.isIndexSelected(index),
+      // TODO removeItem is incorrectly typed. Remove this cast in the fix.
```

Comment:
FYI @Adjective-Object I think this could have been removed as part of https://github.com/OfficeDev/office-ui-fabric-react/pull/8258/commits/25dc76e0fb35318fd8342a280af358e8b27d35f1.

Removing comment now.